ocamlformat.opam

```diff
@@ -35,6 +35,7 @@ depends: [
   "ocp-indent" {with-test}
   "odoc" {>= "1.4.2"}
   "re"
+  "semver2"
```

Comment:
A note for later: Most users are installing OCamlformat with Opam in the same switch as their other dependencies. This may be inconvenient when one of OCamlformat's dependency has a version constraint (which can be added afterward).
This library has a dependency on angstrom < 0.14.0, which is already incompatible with some projects, for example, mrmime, colombe, h2.

Reply:
I'm experimenting something to improve this workflow, so we can have ocamlformat installed independently of what the user has in its current switch. The goal is to not worry about ocamlformat dependencies

Reply:
I wouldn't worry too much, this dependency is easy to delete later if need be.